src/domains/ecommerce-app/api/ecommerce-io/v1/_calculate_fees_policy.xml.tpl

```diff
@@ -1,80 +1,151 @@
+                              <set-url>@($"{{pm-host}}/pp-restapi-CD/v3/paypal/psps")</set-url>
+                              <set-method>GET</set-method>
+                              <set-header name="Authorization" exists-action="override">
+                                  <value>@($"Bearer {(string)context.Variables.GetValueOrDefault("sessionToken","")}")</value>
+                              </set-header>
+                          </send-request>
+                      </when>
+                      <when condition="@(((string)context.Variables["paymentTypeCode"]).Equals("CP") || ((string)context.Variables["paymentTypeCode"]).Equals("BPAY"))">
+                          <send-request ignore-error="true" timeout="10" response-variable-name="getPspForCardsResponse">
+                              <set-url>@($"{{pm-host}}/pp-restapi-CD/v2/payments/{(string)context.Variables["idPayment"]}/psps?idWallet={(string)context.Variables["idWallet"]}&language={(string)context.Variables["language"]}&isList=true")</set-url>
+                              <set-method>GET</set-method>
+                              <set-header name="Authorization" exists-action="override">
+                                  <value>@($"Bearer {(string)context.Variables.GetValueOrDefault("sessionToken","")}")</value>
+                              </set-header>
+                          </send-request>
+                      </when>
+                      <otherwise>
+                          <return-response>
+                              <set-status code="501" reason="Not implemented" />
+                              <set-header name="Content-Type" exists-action="override">
+                                  <value>application/json</value>
+                              </set-header>
+                              <set-body>{
+                                  "title": "Unable to handle payment method",
+                                  "status": 501,
+                                  "detail": "Payment method not handled",
+                              }</set-body>
+                          </return-response>
+                      </otherwise>
+                </choose>
+                <choose>
+                    <when condition="@(((int)((IResponse)context.Variables["getPspForCardsResponse"]).StatusCode) == 200)">
+                        <set-variable name="pmPspsResponse" value="@(((IResponse)context.Variables["getPspForCardsResponse"]).Body.As<JObject>(preserveContent: true))" />
+                        <return-response>
+                            <set-status code="200" reason="OK" />
+                            <set-header name="Content-Type" exists-action="override">
+                                <value>application/json</value>
+                            </set-header>
+                            <set-body>@{
+                                    JArray psps = (JArray)(((JObject)context.Variables["pmPspsResponse"])["data"]);
```

Comment:
Do the PPAY and BPAY/CP APIs have the same response schema? is a only response mapping used for that?